Make player theme selection more intuitive & straightforward

peterklingelhofer opened the pull request:
This makes `Auto` theme one of three equally selectable options, along with `Light` and `Dark`, and the default.

The `Auto` option also now properly detects user system preferences, and if the user previously had `Auto` selected (detected via `localStorage`). If the user is using a dark mode on their system, it now honors their preferences on the initial page load.

This closes https://github.com/resonatecoop/stream/issues/123.

whatSocks commented:
@peterklingelhofer can you post a screenshot (since not all stakeholders run this app locally)

peterklingelhofer commented:
> @peterklingelhofer can you post a screenshot (since not all stakeholders run this app locally) edit: reading the description I don't think a screenshot is super-necessary, but more to reassure the reader that a change doesn't unexpectedly break anything.

Sure thing. I renamed this branch for consistency with current naming conventions elsewhere on this project (which appears to have automatically closed this pull request). I've added back the new pull request with the correct branch name, along with screenshots for every possible user's machine preference/theme selection scenario, here: https://github.com/resonatecoop/stream/pull/128.
